style/client.css

```diff
@@ -1355,6 +1355,9 @@ p.or:after {
 	padding: 0 4px 0 3px;
 	font-style: normal;
 }
+.chat span a small {
```

Comment:
`span` is probably not the correct match. Doesn't `/wall` have a class on that span?

Reply:
This should probably be next to the thing that makes the regular link text gray, too.

Reply:
Ah, I see, it's right below this. This should be moved below `.chat small`, rather than above. Also, the selector should be `.chat .message-announce small`.

Reply:
Hm, the contrast here is kind of poor. Probably the background on announcements should be darker.

Reply:
Should I make the background color of `.message-announce` from battle-log.css darker ?

Reply:
I can handle that, no worries.